conans/test/unittests/model/version/test_version_range.py

```diff
@@ -47,10 +47,45 @@ def test_range(version_range, conditions, versions_in, versions_out):
             assert condition.version == expected_condition[1]
 
     for v in versions_in:
-        assert Version(v) in r
+        assert r.contains(Version(v), None)
 
     for v in versions_out:
-        assert Version(v) not in r
+        assert not r.contains(Version(v), None)
+
+
+@pytest.mark.parametrize("version_range, resolve_prereleases, versions_in, versions_out", [
```

Comment:
Do you know that multiple ``pytest.mark.parametrize`` one on top of the other do the cartesian product? you can reduce these lines 3x with it.
```python
@pytest.mark.parametrize("resolve_prereleases", [True, False, None])
@pytest.mark.parametrize("version_range, versions_in, versions_out", [
```

Reply:
Furthermore, you can also factorize the ``version_range`` and ``versions_in/out``, and it will be even shorter